crowbar_framework/app/models/service_object.rb

```diff
@@ -1543,9 +1561,10 @@ def run_remote_chef_client(node, command, logfile_name)
 
   private
 
-  def wait_for_chef_clients(node)
-    unless RemoteNode.chef_ready?(node, 1200)
-      STDERR.puts "Waiting for already running chef-clients on #{node} failed"
+  def wait_for_chef_clients(node_name)
+    @logger.debug("wait_for_chef_clients: Waiting for already running chef-clients on #{node_name}.")
+    unless RemoteNode.chef_ready?(node_name, 1200)
```

Comment:
@vuntz could we use the [chef_clients_running?](https://github.com/crowbar/barclamp-crowbar/blob/master/crowbar_framework/lib/remote_node.rb#L54) method from `lib/remote_node.rb` instead? or better [chef_ready?](https://github.com/crowbar/barclamp-crowbar/blob/master/crowbar_framework/lib/remote_node.rb#L41) then we have `chef_clients_running?` in a loop